amazon-clone/src/App.vue

```diff
@@ -1,15 +1,19 @@
 <template>
-  <navbar-component />
   
```

Comment:
why there is space

Reply:
I enclosed these under a div as per our screen share you asked me to remove that div hence the space ,no need to worry i removed it in my remote repo ,raising a new PR now with both the changes you suggested.